src/configuration/services/network-storage.md

```diff
@@ -0,0 +1,170 @@
+# Network Storage
+
+Platform.sh supports internal "storage as a service" to provide a file store that can be shared between different application containers.
+
+The network storage service enables a new kind of `mount` that refers to a shared service rather than to a local directory.  Any application can use both `local` or `service` mounts, or either/or, or neither.
+
+
+## Define the service
+
+First, declare a new service in the `services.yaml` file like so:
+
+```yaml
+files:
+    type: network-storage:0.2
```

Comment:
Why does network storage have a version number? This seems confusing.

Reply:
@xtfer is there any that does _not_ have it?

Reply:
I don't know that the system could handle not having one. :smile: I'm more curious if we're going to call it 1.0 before tagging the final.

Reply:
We're probably going to keep 0.2 until it's been battle-tested a bit.